app/uk/gov/hmrc/eoricommoncomponent/frontend/forms/models/subscription/VatDetailsForm.scala

```diff
@@ -17,51 +17,12 @@
 package uk.gov.hmrc.eoricommoncomponent.frontend.forms.models.subscription
 
 import org.joda.time.LocalDate
-import play.api.data.Form
-import play.api.data.Forms._
 import play.api.libs.json.JodaWrites._
 import play.api.libs.json.JodaReads._
-import play.api.data.validation.{Constraint, Invalid, Valid, ValidationError}
 import play.api.libs.json.{Format, Json}
-import uk.gov.hmrc.eoricommoncomponent.frontend.DateConverter
-import uk.gov.hmrc.eoricommoncomponent.frontend.domain.VatIdentification
-import uk.gov.hmrc.eoricommoncomponent.frontend.forms.FormUtils._
-import uk.gov.hmrc.eoricommoncomponent.frontend.forms.FormValidation._
 
-case class VatDetails(postcode: String, number: String, effectiveDate: LocalDate) {
-  def getVatId = VatIdentification(Some("GB"), Some(number))
-}
+case class VatDetails(postcode: String, number: String, effectiveDate: LocalDate)
```

Comment:
Not applicable in Subscription journey

Reply:
The same as BusinessShortName, part of the SubscriptionDetails class. I'll try to remove it in separate commit